tests/cases/fourslash/docCommentTemplateWithMultipleJSDoc3.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path='fourslash.ts' />
+
+/////** @param p */
+/////*/**/

```

Comment:
I don't think this position should generate a template, because the comment on line 3 should apply to `foo` already. At least, I think that's the intent of the code that was previously in jsDoc.ts. @DanielRosenwasser might have a different opinion.

Reply:
Sorry for the confusion caused by my misinterpretation of your previous comment.